### PR TITLE
Countdown to the New Year: 31 Days of Ember Addons - ember-leaflet (Dec 17th 2019)

### PR DESCRIPTION
**Pull request by jayjayjpg**
## What it does

When merged, this will create a new blog post for the DecEmber 2019 blog series.

Should be merged on December 17th, 2019.

### CODE REVIEW

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: ember-leaflet
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber
+alias: 2019/12/17/countdown-to-the-new-year-2019-ember-leaflet
+responsive: true
+---
+
+The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!
+
```
Comment (amyrlam) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: add `READMORE`

Comment (amyrlam) [reply source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: * I guess the proper place based on the other posts pattern is before `What It Does`

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Day 17
+
+In the 17th increment of **Countdown to The New Year** let's take a look at an exciting Ember addon to add **maps** to our apps: **ember-leaflet**!
+
+Learn more about [ember-leaflet on Ember Observer](https://emberobserver.com/addons/ember-leaflet).
+
```
Comment (amyrlam) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: add `READMORE`

(am not sure if this is helpful to repeat, lmk)

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+### What It Does
+
+With ember-leaflet we can create useful maps for our web applications. This addon helps us to integrate the functionality of [Leaflet.js](https://leafletjs.com/) - an open-source JavaScript library for mobile-friendly, interactive maps - into our Ember app.
+
+### Why I Like It
+
+Leaflet.js is already a great library built with simplicity, performance and usability in mind. The addon ember-leaflet makes the developer experience even better for us: we can build complex, multi-layered street maps directly in our templates using the useful layer components provided by the addon API. This makes creating interactive map experiences much more straightforward and much more fun!
+
+Build your own map using ember-leaflet by [checking out the project description on Ember Observer!](https://emberobserver.com/addons/ember-exam)
```
Comment (amyrlam) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]:
```suggestion
Build your own map using ember-leaflet by [checking out the project description on Ember Observer](https://emberobserver.com/addons/ember-exam)!
```

Comment (tylerturdenpants) [reply source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: The link points to ember-exam.

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md (hunk 2 above)
Comment (jayjayjpg) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]:
```suggestion

READMORE
```

Comment (jayjayjpg) [reply source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: Thank you, that's definitely useful - I can update the other posts accordingly as well!

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md (hunk 3 above)
Comment (jayjayjpg) [reply source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: Good catch @amyrlam and @tylerturdenpants - will update in a bit!

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+READMORE
+### What It Does
+
+With ember-leaflet we can create useful maps for our web applications. This addon helps us to integrate the functionality of [Leaflet.js](https://leafletjs.com/) - an open-source JavaScript library for mobile-friendly, interactive maps - into our Ember app.
+
+### Why I Like It
+
+Leaflet.js is already a great library built with simplicity, performance and usability in mind. The addon ember-leaflet makes the developer experience even better for us: we can build complex, multi-layered street maps directly in our templates using the useful layer components provided by the addon API. This makes creating interactive map experiences much more straightforward and much more fun!
+
+Build your own map using ember-leaflet by [checking out the project description on Ember Observer](https://emberobserver.com/addons/ember-exam)!
```
Comment (jayjayjpg) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]:
```suggestion
Build your own map using ember-leaflet by [checking out the project description on Ember Observer](https://emberobserver.com/addons/ember-leaflet)!
```

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md (hunk 1 above)
Comment (jayjayjpg) [reply source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]: Yes, good call - I can update this accordingly :+1:

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: Countdown to The New Year: ember-leaflet
```
Comment (jayjayjpg) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]:
```suggestion
title: Countdown to The New Year: Ember Leaflet
```

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
+title: Countdown to The New Year: Ember Leaflet
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber
+alias: 2019/12/17/countdown-to-the-new-year-2019-ember-leaflet
+responsive: true
+---
+
+The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!
```
Comment (jayjayjpg) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]:
```suggestion
**The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!**
```

File: source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+**The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!**
+
+## Day 17
+
+In the 17th increment of **Countdown to The New Year** let's take a look at an exciting Ember addon to add **maps** to our apps: **ember-leaflet**!
+
+Learn more about [ember-leaflet on Ember Observer](https://emberobserver.com/addons/ember-leaflet).
+
+READMORE
+### What It Does
```
Comment (jayjayjpg) [inline source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md]:
```suggestion

### What It Does
```

**Review by amyrlam — APPROVED** [review]
🗺

**Pull request merged**